platform/src/main/java/org/stellar/anchor/platform/controller/Sep24Controller.java

```diff
@@ -0,0 +1,208 @@
+import java.net.MalformedURLException;
+import java.net.URISyntaxException;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.stellar.anchor.filter.BaseTokenFilter.JWT_TOKEN;
+import static org.stellar.anchor.util.Log.*;
+
+@RestController
+@CrossOrigin(origins = "*")
+@RequestMapping("/sep24")
+public class Sep24Controller {
+  private final Sep24Service sep24Service;
+
+  Sep24Controller(Sep24Service sep24Service) {
+    this.sep24Service = sep24Service;
+  }
+
+  @CrossOrigin(origins = "*")
+  @RequestMapping(
+      value = "/info",
+      method = {RequestMethod.GET})
+  public InfoResponse getInfo() {
+    debug("/info");
+    return sep24Service.getInfo();
+  }
+
+  @CrossOrigin(origins = "*")
+  @RequestMapping(
+      value = "/transactions/deposit/interactive",
+      consumes = {MediaType.APPLICATION_JSON_VALUE},
+      method = {RequestMethod.POST})
+  public InteractiveTransactionResponse deposit(
+      HttpServletRequest request, @RequestBody HashMap<String, String> requestData)
+      throws SepException, MalformedURLException, URISyntaxException {
+    debug("/deposit", requestData);
+    JwtToken token = getSep10Token(request);
+    String fullUrl = getFullRequestUrl(request);
+    InteractiveTransactionResponse itr = sep24Service.deposit(fullUrl, token, requestData);
+    info("interactive redirection:", itr);
+    return itr;
+  }
+
+  @CrossOrigin(origins = "*")
+  @RequestMapping(
+      value = "/transactions/deposit/interactive",
+      method = {RequestMethod.POST},
+      consumes = {MediaType.MULTIPART_FORM_DATA_VALUE})
+  public InteractiveTransactionResponse depositAllType(HttpServletRequest request)
+      throws SepException, MalformedURLException, URISyntaxException {
+    HashMap<String, String> requestData = new HashMap<>();
+    for (Map.Entry<String, String[]> entry : request.getParameterMap().entrySet()) {
+      requestData.put(entry.getKey(), entry.getValue()[0]);
+    }
+
+    return deposit(request, requestData);
+  }
+
+  @CrossOrigin(origins = "*")
+  @RequestMapping(
+      value = "/transactions/withdraw/interactive",
+      consumes = {MediaType.APPLICATION_JSON_VALUE},
+      method = {RequestMethod.POST})
+  public InteractiveTransactionResponse withdraw(
+      HttpServletRequest request, @RequestBody HashMap<String, String> requestData)
+      throws SepException, MalformedURLException, URISyntaxException {
+    debug("/withdraw", requestData);
+    JwtToken token = getSep10Token(request);
+    String fullUrl = getFullRequestUrl(request);
+    InteractiveTransactionResponse itr = sep24Service.withdraw(fullUrl, token, requestData);
+    info("interactive redirection:", itr);
+    return itr;
+  }
+
+  @CrossOrigin(origins = "*")
+  @RequestMapping(
+      value = "/transactions/withdraw/interactive",
+      method = {RequestMethod.POST},
+      consumes = {MediaType.MULTIPART_FORM_DATA_VALUE})
+  public InteractiveTransactionResponse withdrawAllType(HttpServletRequest request)
```

Comment:
This is not all type but form data only:
```suggestion
  public InteractiveTransactionResponse withdrawFormData(HttpServletRequest request)
```

Reply:
Renamed